src/templates/items/parts/item-status.hbs

```diff
@@ -6,6 +6,14 @@
                     <input type="checkbox" name="system.proficient" {{checked itemData.proficient}}/> {{ localize "SFRPG.Items.Proficient" }}
                 </label>
                 {{/if}}
+                {{#if isWeapon}}
+                <label class="checkbox">
+                    <input type="checkbox" name="system.isEquipment" {{checked itemData.isEquipment}}/> {{ localize "SFRPG.Items.Weapon.IsEquipment" }}
```

Comment:
It'd be nice to have a tooltip on this describing the use case and behaviour of this feature

Reply:
Agreed, I've added one with a description of what checking it signifies